crates/bevy_pbr/src/material.rs

```diff
@@ -502,6 +532,12 @@ pub fn queue_material_meshes<M: Material>(
             let Some(material) = render_materials.get(material_asset_id) else {
                 continue;
             };
+
+            let forward = match material.properties.render_method {
+                OpaqueRendererMethod::Auto | OpaqueRendererMethod::Forward => true,
```

Comment:
don't you need to check the resource here? i think this is effectively disabling deferred for auto materials.

Reply:
Nope, that happens here: https://github.com/DGriffin91/bevy/blob/9e4d37f010bb9caeeed76617282c334a30a2414a/crates/bevy_pbr/src/material.rs#L841

Reply:
could we `=> unreachable!()` the Auto case then please

Reply:
Yep!